app/Handlers/Events/SendFirstVoteMessage.php

```diff
@@ -39,21 +35,38 @@ public function handle(UserCastFirstVote $event)
             'candidate_name' => $event->candidate->name,
 
```

Comment:
Please add `candidate_id => $event->candidate->id` in addition to the tag setting.